tests/test_pipeline/test_base.py

```diff
@@ -8,9 +8,392 @@
 
 from etna.datasets import TSDataset
```

Comment:
Do we have some integration tests for backtest with FoldMask?

Reply:
In `test_pipeline` there are tests that use `FoldMask`. But there aren't tests that use `FoldMask` with integer timestamp. It will be fixed in the #221 .